src/components/generic/Modal/Modal.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import {Modal, ModalBody, ModalHeader} from '@momentum-ui/react';
+import {WEBEX_COMPONENTS_CLASS_PREFIX} from '../../../constants';
+
+/**
+ * Modal component
+ *
+ * @param {object} props  Data passed to the component
+ * @param {string} props.className   Custom CSS class to apply
```

Comment:
too many spaces:)

Reply:
for optional props, please use the square bracket syntax: `[props.className]`, see https://jsdoc.app/tags-param.html#optional-parameters-and-default-values